src/lib/map-factory.js

```diff
@@ -1,9 +1,14 @@
+import ported from "./object-mapper/object-mapper";
+import original from "object-mapper";
 import Mapper from "./mapper";
 
-export default function createMapper() {
+export default function createMapper(options) {
+
+  options = options || {};
```

Comment:
Just one more thing... eslint would fail on this (with the default setting) because a passed in parameter is modified. And I think he's right. Here the passed in parameter just a pointer which points to something already created in the heap when an other scope was generated. So you're are modifying something which might be used something else somewhere else. I would create a new variable in this scope...

```
const opts = options || {};
```
I believe this is called side effect. I might be wrong though...